app/assets/javascripts/wikis.js

```diff
@@ -26,9 +26,9 @@ function setupWiki(node_id, title, raw, logged_in) {
     });
     $('#content').hide();
   } else {
-    $('#content').html(shortCodePrompt($('#content')[0], {
-      submitUrl: '/wiki/replace/' + node_id
-    }));
+    // $('#content').html(shortCodePrompt($('#content')[0], {
```

Comment:
Do we need this?

Reply:
Um, let me look at this code to see what it's supposed to do!

Reply:
OK, strangely I can't find this code in our codebase but I definitely wrote it.

OMG ok i chased this SUPER DEEP - back to https://github.com/publiclab/plots2/pull/1224/files

where I had made this library:

https://github.com/jywarren/short-code-forms

whoaaaaa

It looks for: `[prompt:_______]` style inline tags: https://github.com/jywarren/short-code-forms/blob/master/examples/index.html#L39

The code that drives this is not complex: https://github.com/jywarren/short-code-forms/blob/master/dist/short-code-prompts.js

I think we do need this, but i'm not sure why this is causing trouble because it's supposed to look for the word "prompt":

`/\[prompt:(\w*):*([\s\w]*):*(\w*)\]/g;` is the regex. 

Strange!

Reply:
I'm not sure how you're testing it but could this help to debug?

Reply:
Oh cool, I was just looking on how /wiki code is different from /note and found this to be different. I commented this code and the maps are now working!